exercises/matrix/example.exs

```diff
@@ -0,0 +1,63 @@
+defmodule MatrixStruct do
```

Comment:
I just added a PR to rename the `Matrix` class in saddle-points to `SaddlePoints` (https://github.com/exercism/xelixir/pull/332), so if that's accepted, I can rename this to `Matrix`.